test/unit/concerns/orchestration/compute_test.rb

```diff
@@ -22,29 +22,48 @@ class OrchestrationComputeTest < ActiveSupport::TestCase
       @host.send(:setAttachIsoImage)
     end
 
-    test 'provisioning a host with provision method bootdisk should queue bootdisk tasks' do
+    test 'provisioning a host with provision method bootdisk should detach iso' do
```

Comment:
Is this test any good?

Reply:
added this one because there were 2 tests (iso_upload, iso_attach) above testing the methods directly.